packages/devextreme/js/__internal/grids/grid_core/data_controller/m_data_controller.ts

```diff
@@ -1218,6 +1218,7 @@ export class DataController extends DataHelperMixin(modules.Controller) {
   public updateItems(change?, isDataChanged?) {
     change = change || {};
     const that = this;
+    change.isDataValueChanged = that.changed._fired;
```

Comment:
I recommend using a public fired method. I also suggest renaming the flag to isFirstRender/isFirstChanged:
`change.isFirstRender = !that.changed.fired();`